main.py

```diff
@@ -0,0 +1,51 @@
+originArray = [[1,2,3],[4,5,6],[7,8,9]]
```

Comment:
General CR comment: You might want to use PEP8 and pylint tools before handing me the exercises.
It will teach you how to write code following the PEP 8 Python conventions.
1) Avoid using Hungarian Notation, in which you specify the type of the variable in the variable name.
2) Use lowercase_with_underscores for variable and function names in Python, never camelCase.
3) Almost always put space after comma.
Therefor, in my opinion, the line should look like this:
```origin_input = [[1, 2, 3], [4, 5, 6], [7, 8, 9]]```
Try to fix these problems in the rest of the program. :)
To install the pep8 and pylint tools, use: `pip insall pep8 pylint`
Use this tools using: `pep8 file.py` and `pylint file.py`.
Not all the errors are must-follow. Feel convenient to ask me anything about that.